src/Symfony/Bundle/FrameworkBundle/Tests/DependencyInjection/Compiler/AddCacheWarmerPassTest.php

```diff
@@ -33,7 +33,6 @@ public function testThatCacheWarmersAreProcessedInPriorityOrder()
             ->will($this->returnValue($services));
         $container->expects($this->atLeastOnce())
             ->method('getDefinition')
-            ->with('cache_warmer')
```

Comment:
does this losen the assertion of the test to you?
imho it's a "we shouldn't care" internal detail - dealing with it is not related to the business of this test case (same for other changes)